Gemfile

```diff
@@ -13,23 +13,24 @@ gem 'rgl', '~> 0.5.7'
 gem 'safe_yaml', '~> 1.0'
 gem 'thor', '~> 1.1.0'
 gem 'toml', '~> 0.2.0'
+gem 'license_finder'
 
 group :scanners do
   gem 'brakeman', '4.10.0'
   gem 'bundler-audit', '~> 0.8.0'
 end
 
```

Comment:
be sure to restore the original groups

Reply:
please add the version for license finder, like the other gems